CHANGELOG.md

```diff
@@ -8,6 +8,8 @@ Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
 
+## [3.12.0] - 2020-11-24
+
```

Comment:
Maybe 
"Add dash endpoint to mediapackage channel" 
is part of 
"Manage live streaming using AWS Elemental"

Reply:
Yes definitely